### DISCUSSION TITLE
Translation files

### DISCUSSION THREAD

**Discussion by lxhom** [fnuecke/oc2#125] [Q&A]
Hi! I've been working on a German translation and so far I have translated `/src/main/resources/assets/oc2/lang/de_de.json` and started with the markdown files at `/src/main/resources/assets/oc2/doc/de_de/`, are there more files that need to be translated and/or do I have to register the translation somewhere?

**Comment by CoolCat467** [comment]
I don't believe there is anything else to do other than just doc and lang which you've got, so unless I'm missing something I think you're good.
